controllers/history.controller.js

```diff
@@ -0,0 +1,81 @@
+const userDb = require("../models/user.model");
+
+module.exports.getHistory = async (req, res) => {
+  const { userId } = req.params;
+  try {
+    const user = await userDb.findById(userId);
+    const data = await user.execPopulate("history");
+    return res.status(200).json({ success: true, data: [...data.history] });
```

Comment:
Keeping your json responses consistent would help when you're fetching/working with the data! Most of the previous responses have a message, adding one here would be good!